ONSdigital/dp-frontend-renderer
# Remove dataset preview

### What

- Add feature flag if conditional around dataset preview block

### How to review

- Pull https://github.com/ONSdigital/dp-frontend-filter-dataset-controller/pull/180
- Run `dp-frontend-filter-dataset-controller` with `ENABLE_DATASET_PREVIEW=false` set. 
- Filter a dataset and proceed to the output page. **No** preview should show

### Who can review

Anyone
